tools/gen_stubs.zig

```diff
@@ -0,0 +1,31 @@
+const std = @import("std");
+
+// Example usage:
+// objdump --dynamic-syms /path/to/libc.so | ./gen_stubs > lib/libc/musl/libc.s
+pub fn main() !void {
+    const stdin = std.io.getStdIn().reader();
+    const stdout = std.io.getStdOut().writer();
+
+    // This is many times larger than any line objdump produces should ever be
+    var buf: [4096]u8 = undefined;
+
+    // Sample input line:
+    // 00000000000241b0 g    DF .text	000000000000001b copy_file_range
+    while (try stdin.readUntilDelimiterOrEof(&buf, '\n')) |line| {
+        // the lines we want all start with a 16 digit hex value
+        if (line.len < 16) continue;
+        _ = std.fmt.parseInt(u64, line[0..16], 16) catch continue;
+
+        // the last whitespace-separated column is the symbol
+        var symbol: []const u8 = undefined;
+        var it = std.mem.tokenize(line, &std.ascii.spaces);
+        while (it.next()) |s| symbol = s;
+
+        try stdout.print(
+            \\.globl {s}
+            \\.type {s}, @function;
+            \\{s}:
+            \\
+        , .{ symbol, symbol, symbol });
```

Comment:
This is where numbered/named fmt parameters shine :) no more `symbol, symbol, symbol`

Reply:
Oh I am very much looking forward to them, especially for my [code generator](https://github.com/ifreund/zig-wayland/blob/52326e7ee09d7acb6b55855f7a697af083ae973a/src/scanner.zig#L367).